src/utils/decorators/input/input.js

```diff
@@ -319,7 +319,13 @@ const Input = ComposedComponent => class Component extends ComposedComponent {
       // renders a fake input - useful for screens with lots of inputs
       const classes = classNames(this.inputProps.className, 'common-input__input--fake');
       input = (
-        <div className={ classes } onMouseOver={ this.inputProps.onMouseOver }>
+        <div
+          className={ classes }
+          onBlur={ this.inputProps.onBlur }
+          onFocus={ this.inputProps.onFocus }
+          onMouseOver={ this.inputProps.onMouseOver }
+          onMouseOut={ this.inputProps.onMouseOut }
```

Comment:
what are these for?

Reply:
This was actually added as a fake input for a performance boost with minimal event listeners etc applied. Can we get away with just adding the `onFocus` to pair with the `onMouseOver`?

Should the `onFocus` and `onMouseOver` call the same function?

Reply:
Because this is a decorator, I had thought the intention was to allow the developer to define any of these handlers on the input itself and they would get passed into the decorator (currently they would be thrown away except for the `onMouseOver`).

Reply:
Yeah but this isn't an input, it's a fake input (a div) with the appearance of an input. It won't actually have a focus or blur. The purpose of it was to render something really simple for a performance boost (it was originally used in Partner Edition during the import process). I think it only needs the mouse over event.

Reply:
I've removed the changes from this PR 👍